wave-app/src/descriptions.py

```diff
@@ -0,0 +1,51 @@
+FIRMS_INFO = '''
```

Comment:
since this is not a source code file and only text, you could add it in the `models/` directory as a txt file and read it. For example, how model file(s) and HTML are loaded from `models/`